app/commands/v2/represent_downstream.rb

```diff
@@ -48,7 +48,7 @@ def downstream_live(content_id, locale)
             content_id: content_id,
             locale: locale,
             payload_version: event.id,
-            message_queue_update_type: "links",
+            message_queue_event_type: "links",
```

Comment:
cool - can we get a card in the backlog for it? (with any information as to what would block the change)